.idea/modules.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/.idea/windup_cli_tests.iml" filepath="$PROJECT_DIR$/.idea/windup_cli_tests.iml" />
+    </modules>
+  </component>
+</project>
```

Comment:
No newline at end of file

Reply:
Solved in 6287c0df3c68ff7640dc8ed34bb0fc9c76cb7152.